Src/unstable/library/network/openssl/src/crypto/ciphers/wrapper/ssl_crypto_externals.e

```diff
@@ -451,29 +466,83 @@ feature -- Access
 		end
 
 	c_evp_digestsigninit (a_ctx: POINTER; a_pctx: POINTER; a_type: POINTER; a_engine: POINTER; a_key: POINTER): INTEGER
-			-- --https://www.openssl.org/docs/man1.1.0/crypto/EVP_DigestSignInit.html
+			-- https://www.openssl.org/docs/man1.1.0/crypto/EVP_DigestSignInit.html
```

Comment:
you could also use EIS ... but comment is also fine for now.

Reply:
yes, I was lazy and I was not sure if add it to the feature level or at class level